### PR TITLE
fix(curriculum): update challenge tests to pass a valid solution

### PR DESCRIPTION
**Pull request by yossirise**
…a string

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-development-libraries/react/add-inline-styles-in-react.md [hunk 1]
```diff
@@ -33,7 +33,10 @@ assert(styles.color === 'purple');
 The `styles` variable should have a `fontSize` property set to a value of `40`.
 
 ```js
-assert(styles.fontSize === 40);
+assert(
+  styles.fontSize === 40 ||
+  styles.fontSize === '40'
+);
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/03-front-end-development-libraries/react/add-inline-styles-in-react.md:39]:
```suggestion
assert(styles.fontSize == 40);
```

File: curriculum/challenges/english/03-front-end-development-libraries/react/add-inline-styles-in-react.md [hunk 2]
```diff
@@ -61,7 +64,8 @@ assert(
     const mockedComponent = Enzyme.shallow(React.createElement(Colorful));
     return (
       mockedComponent.props().style.color === 'purple' &&
-      mockedComponent.props().style.fontSize === 40 &&
+      (mockedComponent.props().style.fontSize === 40 ||
+        mockedComponent.props().style.fontSize === '40') &&
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/03-front-end-development-libraries/react/add-inline-styles-in-react.md:68]:
```suggestion
      mockedComponent.props().style.fontSize == 40 &&
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you, for taking the time to contribute.

Please see the requested changes

**Review by naomi-lgbt — APPROVED** [review]
This is one of those cases where `"40"` *works* and should probably pass, but I don't think it's something most people would write. 🤔 

Anyway, happy to land this.

**Review by ShaunSHamilton — APPROVED** [review]
Thank you, for making those changes

**Comment by ShaunSHamilton** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**